ttnn/ttnn/device.py

```diff
@@ -138,11 +138,37 @@ def is_grayskull(device):
     return device.arch() == ttnn._ttnn.device.Arch.GRAYSKULL
 
 
+def is_blackhole(device):
+    return device.arch() == ttnn._ttnn.device.Arch.BLACKHOLE
+
+
 SetDefaultDevice = ttnn._ttnn.device.SetDefaultDevice
 GetDefaultDevice = ttnn._ttnn.device.GetDefaultDevice
 format_input_tensor = ttnn._ttnn.device.format_input_tensor
 format_output_tensor = ttnn._ttnn.device.format_output_tensor
 pad_to_tile_shape = ttnn._ttnn.device.pad_to_tile_shape
 
 
+def GetComputeKernelConfig(
+    math_fidelity=ttnn.MathFidelity.LoFi,
+    math_approx_mode=True,
+    fp32_dest_acc_en=False,
+    packer_l1_acc=False,
+    dst_full_sync_en=False,
+):
+    device = ttnn.GetDefaultDevice()
+    if is_wormhole_b0(device) or is_blackhole(device):
+        return ttnn.WormholeComputeKernelConfig(
```

Comment:
I would create blackhole config here if it's blackhoel arch.
I know that currently they are just aliased, but it doesn't mean it will stay that way.

Reply:
Done.